cmd/podman/play/kube.go

```diff
@@ -150,6 +153,11 @@ func kube(cmd *cobra.Command, args []string) error {
 	if kubeOptions.Down {
 		return teardown(yamlfile)
 	}
+	if kubeOptions.Replace {
+		if err := teardown(yamlfile); err != nil {
```

Comment:
Should this succeed if a previous podman play kube had not run?  IE ignore Kube does not exists errors?